angular-lib/src/angular/ui/components/alerts/alert.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, OnInit, OnDestroy, Input, ViewEncapsulation } from '@angular/core';
+import { Router, NavigationStart } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+import { Alert, AlertType } from './alert.model';
+import { AlertService } from './alert.service';
+
+@Component({
+  selector: 'cloudapp-alert',
+  templateUrl: 'alert.component.html',
+  styleUrls: ['alert.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
I agree. But since the alerts are added dynamically, the styles are not applied unless encapsulation is off. I added `cloudapp-alert` as a parent selector in the CSS to be sure the styles are limited to this component.